k8s/jiva/snapshot-cleanup.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+usage()
+{
+    echo "Usage: bash script.sh <controller_pod_name> <application_namespace> <numbber_of_snapshots_to_delete>"
+    echo "WARNING: This script need to be run from client that have access to the cluster. This script can run for longer time based on the amount of data in the snapshots that need to be deleted. Please don't cancel the operation while it is running. If the script gets broken due to any network issue, you need to run unblock_service.sh script"
+    exit 1
+}
+
+delete_jiva_snapshot()
+{  
+    min_required_snapshot=4
+	ctrl_pod_name=$1
+	application_namespace=$2
+	number_of_snapshot=$3
+	count=0
+    cat patch.json > tmp.json
+	snapshot_number_cmd='kubectl exec -it $ctrl_pod_name -n $application_namespace -- jivactl snapshot ls | grep -v ID | wc -l'
+	snapshot_name_cmd='kubectl exec -it $ctrl_pod_name -n $application_namespace -- jivactl snapshot ls | grep -v ID | tail -1'
+
+	if [ $(eval $snapshot_number_cmd) -le $((number_of_snapshot-1)) ]; then
+		echo "Provided number of snapshots are not present"
```

Comment:
putting a return here will make the readability better. avoids further deep nested blocks below.